bndtools.m2e/lifecycle-mapping-metadata.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lifecycleMappingMetadata>
+	<pluginExecutions>
+		<pluginExecution>
+			<pluginExecutionFilter>
+				<groupId>biz.aQute.bnd</groupId>
+				<artifactId>bnd-maven-plugin</artifactId>
+				<versionRange>[3.0.0,)</versionRange>
+				<goals>
+					<goal>bnd-process</goal>
+				</goals>
+			</pluginExecutionFilter>
+			<action>
+				<configurator>
+					<id>bndtools.m2e.bndconfigurator</id>
+				</configurator>
+			</action>
+		</pluginExecution>
+	</pluginExecutions>
+</lifecycleMappingMetadata>
```

Comment:
I just wanted git to add the `src` folder so eclipse would not fail the build and add error markers to bnd.bnd.  I could have used `.gitkeep` or `.touch` as other marker files.  I just used `.gitignore`.